bears/verilog/VerilogLintBear.py

```diff
@@ -24,6 +24,7 @@ class VerilogLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Simplification', 'Syntax'}
```

Comment:
well it detects uf you specify stuff and don't use it, that's unused code detection and should be listed IMO